docs/HowTo/Find-and-Connect/Specifying-NAT.md

```diff
@@ -24,19 +24,10 @@ device.
     UPnP support is often disabled by default in networking firmware. If disabled by default, you
     must explicitly enable UPnP support.
 
-!!! important
-
-    When the NAT method is set to `UPNP`, the advertised port is the same as the
-    [listening port](../../Reference/CLI/CLI-Syntax.md#p2p-port).
-
 ## None
 
 Specify `NONE` to explicitly configure the external IP address and ports advertised using
-[`--p2p-host`](../../Reference/CLI/CLI-Syntax.md#p2p-host) and
-[`--p2p-port`](../../Reference/CLI/CLI-Syntax.md#p2p-port) for the P2P service.
-
```

Comment:
sorry, so p2p-port is still the local listening port, just not the advertised, it's only p2p-host that's not valid for teku

Reply:
actually i think thats ok